e2e-tests/cypress/tests/integration/channels/insights/insights_spec.ts

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+// Stage: @prod
+
+describe('Insights', () => {
+    let teamA;
+
+    before(() => {
+        cy.shouldHaveFeatureFlag('InsightsEnabled', true);
+
+        cy.apiInitSetup().then(({team}) => {
+            teamA = team;
+        });
+    });
+    it('Check all the cards exist', () => {
+        cy.apiAdminLogin();
+
+        // # Go to the Insights view
+        cy.visit(`/${teamA.name}/activity-and-insights`);
+
+        // * Check top channels exists
+        cy.get('.top-channels-card').should('exist');
```

Comment:
`.should('exist')` isn't necessary (is a default assertion on `cy.get`,`.find`, etc.)